src/themes/default/styles.scss

```diff
@@ -78,220 +69,17 @@ $duration: 200ms;
-        border: 0;
-        border-radius: $message-border-radius-small;
-        width: 100%;
-        margin: {
-            bottom: $message-spacing-thin;
-        }
-
-        &:last-of-type {
-            border-bottom-left-radius: $message-border-radius;
-        }
-    }
-
-    .slick-slider {
-        position: relative;
-        .slick-prev {
-            position: absolute;
-            border: 0;
-            left: $message-spacing-thick * -1;
-            width: $message-spacing-thick;
-            top: 0;
-            height: 100%;
-            background-color: $grey-light;
-
-            color: $white;
-            transition: 0.2s ease;
-            z-index: 1;
-
-            outline: none;
-            border-top-right-radius: $message-border-radius-small;
-            border-bottom-right-radius: $message-border-radius-small;
-            cursor: pointer;
-            &:hover {
-                width: 2rem;
-                background-color: $grey-dark;
-            }
-        }
-        .slick-next {
-            @extend .slick-prev;
-            right: $message-spacing-thick * -1;
-            left: auto;
-            border-top-right-radius: 0px;
-            border-bottom-right-radius: 0px;
-            border-top-left-radius: $message-border-radius-small;
-            border-bottom-left-radius: $message-border-radius-small;
-        }
-        .slick-dots {
-            list-style: none;
-            padding-left: 0;
-            margin: 0 auto;
-            text-align: center;
-            li {
-                display: inline-block;
-                button {
-                    background-color: $grey-light;
-                    border: 0;
-                    color: $grey-light;
-                    outline: none;
-                    transition: 0.2s ease;
-                    cursor: pointer;
-                    border-radius: 0.5rem;
-                    width: 1rem;
-                    height: 1rem;
-                    margin: $message-spacing-thin;
-                }
-
-                &.slick-active {
-                    button {
-                        background-color: $grey-dark;
-                        color: $grey-dark;
-                    }
-                }
-            }
-        }
-    }
-}
-
-.ButtonContainer {
-    width: 109%;
-    margin-top: $rhythm * 2;
-    margin-left: $rhythm * -2;
-    margin-bottom: $rhythm * 2;
-    overflow-x: auto;
-    overflow-y: hidden;
-    white-space: nowrap;
-
-    .Button {
-        display: inline-block;
-    }
-}
+$avatar-size: $rhythm * 8;
 
 @import './_animations.scss';
-
+@import './_containers.scss';
+@import './_message-list.scss';
 @import './components/Avatar/_avatar.scss';
 @import './components/Button/_button.scss';
 @import './components/Input/_input.scss';
 @import './components/Image/_image.scss';
 @import './components/Menu/_menu.scss';
 @import './components/Text/_text.scss';
+@import './components/Carousel/_carousel.scss';
```

Comment:
@Pringels I moved this into components folder since this might be components in the future. Please let me know if you want me to move it back.